recipes/pvWebMonitor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pvWebMonitor" %}
+{% set version = "2021.0.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f8b4cf98cdc4ee92e1e95f649944821816c0e05624b99f2de26d9f20932c358
+
+build:
+  entry_points:
+    - pvWebMonitor=pvWebMonitor.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pyepics
+    - lxml
+    - numpy
+    - ophyd
+    - packaging
+
+test:
+  imports:
+    - pvWebMonitor
+  commands:
+    - pip check
+    - pvWebMonitor --help
+  requires:
+    - pip
+
+about:
+  home: http://bcda-aps.github.io/pvWebMonitor/
+  summary: Create static web site from EPICS PVs.
+  license: BSD-4-Clause
```

Comment:
@conda-forge/core Is `BSD-4-Clause` OK for us? I know in some other cases that the original advertising clause made issues.

Reply:
See feedstock for related package `apstools`: https://github.com/conda-forge/apstools-feedstock/blob/352d3253def51788ffb17212c6b5244decff5cc0/recipe/meta.yaml#L67-L72

Reply:
I'm reading the BSD-4-Clause license https://spdx.org/licenses/BSD-4-Clause.html

I see the following notable difference in the text located in https://github.com/BCDA-APS/pvWebMonitor/blob/main/src/pvWebMonitor/LICENSE

> Software changes, modifications, or derivative works, should be noted with comments and the author and organization's name.


> 4. The software and the end-user documentation included with the  redistribution, if any, must include the following acknowledgment: "This product includes software produced by UChicago Argonne, LLC  under Contract No. DE-AC02-06CH11357 with the Department of Energy.

Is not part of the BSD-4-Clause license linked above from SPDX.

Clause 3 from SPDX link
> 3. All advertising materials mentioning features or use of this software must display the following acknowledgement: This product includes software developed by the organization. 

is notably missing.

To me, given the simplicity of the BSD licenses, I do think that this is a BSD-4-Clause license. To me, the license should be listed as "Other"

Reply:
Fine by me.

Reply:
The requirement for the "end user documentation" to include that notice gives me pause. I'm not sure what constitutes documentation. I'm not able to help @xhochy get to the answer of his original question.

Reply:
Searching now for a match on https://spdx.org/licenses/ since `Other` is not an allowed term.  The language in the license file is supplied by the lab's lawyers and does not give flexibility to change anything other than the name of the software, the software group, and the copyright years.

Reply:
The [Lawrence Berkeley National Labs BSD variant license](https://spdx.org/licenses/BSD-3-Clause-LBNL.html) is substantially identical through clause 3.  The addition of clause 4 makes this license a variation.

Reply:
`BSD-3-Clause-LBNL`

Reply:
[BSD-4-Clause-UC](https://spdx.org/licenses/BSD-4-Clause-UC.html) is similar to this license with clauses 3 & 4 swapped (and this is not UC, of course).

Reply:
[EPICS Open License](https://spdx.org/licenses/EPICS.html) is an SPDX-recognized license used by other software in this community.  I'll look into shifting to that license.

Reply:
> * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

[_Au contraire_](https://spdx.org/licenses/EPICS.html), `EPICS` is a recognized SPDX identifier.  I'll change it to `EPICS Open License` anyway.

Reply:
You can also create your custom license using a LicenseRef: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions